Portfolio/index.css

```diff
@@ -1 +1,53 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+.main{
+    padding: 10px;
+    box-sizing: border-box;
+    background-image: radial-gradient( circle farthest-corner at 10% 20%,  rgba(255,94,247,1) 17.8%, rgba(2,245,255,1) 100.2% );
+}
+
+
+
+.asset{
+    height: 200px;
+    width: 200px;
+}
+
+.footer{
+    border-top: 2px solid white;
+    margin-top: 10px;
+}
+
+.form{
+    width: 50%;
+    place-items: center;
+    backdrop-filter: blur(4px);
+    border: 2px solid white;

```

Comment:
Consider adding some spacing between the property and value for better readability.